doc/user/bgp.rst

```diff
@@ -2097,6 +2079,34 @@ is called as named community lists.
    This command defines a new expanded community list. The argument to
    (100-199) defines the list identifier.
 
+.. _bgp-community-alias:
+
+Community alias
+^^^^^^^^^^^^^^^
+
+BGP community aliases are useful to quickly identify what communities are set
+for a specific prefix in a human-readable format. Especially handy for a huge
+amount of communities. Accurately defined aliases can help you faster spot
+ things on the wire.
+
+.. clicmd:: bgp community alias NAME ALIAS
+
+   This command creates an alias name for a community that will be used
+   later in various CLI outputs in a human-readable format.
+
+   .. code-block:: frr
+
+      ~# show ip bgp 172.16.16.1/32
```

Comment:
IMO, it's quite clear with the command's definition `.. clicmd:: bgp community alias NAME ALIAS`.

Reply:
Sure, it's clear how to use the command, but it's not clear how the output is affected. Is that `community-1` and `lcommunity-1` strings? It would be more clear if you add `bgp community alias 65001:14 community-1` and `bgp community alias 65001:123:1` before the `show`. Anyway, it's not a blocking comment if you don't want to do this.

Reply:
Added.